src/main/java/gov/usds/case_issues/controllers/CaseDetailsApiController.java

```diff
@@ -70,7 +70,11 @@ public CaseDetails getCaseDetails(@PathVariable String caseManagementSystemTag,
 	@PostMapping("activeSnooze/notes")
 	public ResponseEntity<?> addNote(@PathVariable String caseManagementSystemTag,
 			@PathVariable String receiptNumber, @RequestBody NoteRequest newNote) {
-		_caseDetailsService.annotateActiveSnooze(caseManagementSystemTag, receiptNumber, newNote);
+		try {
+			_caseDetailsService.annotateActiveSnooze(caseManagementSystemTag, receiptNumber, newNote);
+		} catch (IllegalArgumentException e) {
+			return ResponseEntity.badRequest().build();
```

Comment:
If we can get this exception catch stashed in the ControllerAdvice, we will be happier.